app/src/main/res/layout/activity_main.xml

```diff
@@ -5,6 +5,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:fitsSystemWindows="true"
```

Comment:
I'm *hoping* this won't cause issues with the legacy build, but may have to pull it out from this branch if we see problems.